jekyll/_cci2/test.md

```diff
@@ -1,51 +1,88 @@
 ---
 layout: classic-docs
-title: "Test"
-description: "CircleCI test automation setup"
+title: "Automated testing in CircleCI"
+description: "An overview of setting up testing automation, integration, and analytics"
 version:
 - Cloud
 - Server v3.x
 - Server v2.x
 ---
 
-Refer to the following video and documents for help with setting up your tests.
+## Introduction
+{: #introduction }
 
-## How to build, test, and deploy video tutorial
-{: #how-to-build-test-and-deploy-video-tutorial }
+CircleCI allows you to automatically test your code before changes are merged. You can integrate testing tools and frameworks such as Jest, Mocha, pytest, JUnit, Selenium, XCTest, and more.
 
-Watch the following video for a detailed tutorial of Docker, iOS, and Android builds.
-<div class="video-wrapper">
-  <iframe width="560" height="315" src="https://www.youtube.com/embed/Qp-BA9e0TnA" frameborder="0" allowfullscreen></iframe>
-</div>
+When you integrate your tests into your CircleCI pipelines, you not only deliver software to your users more reliably and efficiently, you get feedback more quickly so you can fix issues and failed tests faster. If you save your test data in CircleCI, you can also take advantage of the Test Insights as well as parallelism features to identify opportunities to further optimize your pipelines.
```

Comment:
I would add: you can debug your tests by viewing test output data for each failed test (or something like that)